contracts/WrappedEditions.cdc

```diff
@@ -88,6 +88,10 @@ pub contract WrappedEditions {
             return self.supply
         }
 
+        pub fun getRaffleID(): UInt64 {
```

Comment:
Is this safe?

Reply:
In the testing framework itself, you can just look for emitted events, i would take that approach instead of adding more methods here

https://github.com/Flowtyio/flow-raffle/blob/main/test/FlowtyRaffles_tests.cdc#L50

Reply:
Okay, thanks will do!

Reply:
Okay, that is much easier too. Didn't think about getting the raffleID from event.